build/common.props

```diff
@@ -6,7 +6,7 @@
 
   <!-- When building on non-Windows machines, we need the reference assemblies to build full-framework assemblies. -->
   <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework' AND '$(DotNetBuildFromSource)' == 'true'">
-     <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0-alpha-5" PrivateAssets="All" />
+     <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0-preview.1" PrivateAssets="All" />
```

Comment:
Why do we still need this if they do not build .NET Framework anymore? 

This was specifically added by source-build folks. 

See: https://github.com/NuGet/NuGet.Client/commit/7be2fe5b0d1fed0a67dc69a8082c14f9b616e9b0#diff-aa45b664821154d7562b7b6575d05bc6

//cc @crummel

Reply:
Yes, this can go away.  We are no longer building .NET FX in source-build's NuGet.  /cc @NikolaMilosavljevic

Reply:
Ok, I will remove the PackageReference and test this change against dotnet/source-build